web/src/app/alerts/AlertsList.js

```diff
@@ -106,7 +107,7 @@ function getStatusFilter(s) {
 export default function AlertsList(props) {
   const classes = useStyles()
   const width = useWidth()
-  const isFullScreen = isWidthDown('md', width)
+  const isMobileScreenSize = isWidthDown('md', width)
```

Comment:
💯 better var name (I **always** thought this was opposite)

Reply:
Yeah, it's 100% always confusing to me, too! Thankfully with things changing to `width: md` in hooks it will eventually go away all together